src/backend/joanie/tests/core/test_api_contract.py

```diff
@@ -155,12 +348,7 @@ def test_api_contract_read_authenticated(self):
         content = json.loads(response.content)
         self.assertEqual(
             content,
-            {
-                "id": str(owned_contract.id),
-                "definition": str(owned_contract.definition.id),
-                "order": str(owned_contract.order.id),
-                "signed_on": None,
-            },
+            self._return_from_contract(owned_contract),
```

Comment:
I think it's better to have the full dict here. Otherwise reading tests to understand why they fail in a refactoring will be very difficult.